src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -39,6 +39,29 @@
   to `null` will now result in an `ArgumentNullException` being thrown.
   ([#5434](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5434))
 
+* Added `IOpenTelemetryBuilder.UseOtlpExporter` extension to simplify setup of
+  the OTLP Exporter when all three signals are used (logs, metrics, and traces).
+  The new extension has the following behaviors:
+
+  * Calling `UseOtlpExporter` will automatically enable logging, tracing, and
+    metrics. Additional calls to `WithLogging`, `WithMetrics`, and `WithTracing`
```

Comment:
> Additional calls to `WithLogging`, `WithMetrics`, and `WithTracing` are NOT required.

Do we configure the TracerProvider and MeterProvider to listen to any `ActivitySource` or `Meter` for the export to export something (when there are no `WithTracing` and `WithMetrics` calls)?

Reply:
Not currently. So there will likely need to be calls to either `WithTracing` / `WithMetrics` or `ConfigureOpenTelemetryTracerProvider` / `ConfigureOpenTelemetryMeterProvider` to turn _something_ on. For metrics users could use the config stuff @noahfalk added in .NET 8 to eliminate those calls.

I tweaked the CHANGELOG a bit: https://github.com/open-telemetry/opentelemetry-dotnet/pull/5400/commits/0faac1a49f79c5b906d5b8d854bc651ef7db9ede

LMK what you think about that.

Reply:
> Automatically turns on Logging, Metrics, & Tracing

Could you update the PR description as well to mention this?

Reply:
Done